pkgs/data/fonts/tewi/default.nix

```diff
@@ -0,0 +1,47 @@
+{stdenv, fetchgit, bdftopcf, mkfontdir, mkfontscale}:
+
+stdenv.mkDerivation rec {
+  date = "2015-03-08";
+  name = "tewi-font-${date}";
+
+  src = fetchgit {
+    url = "https://github.com/lucy/tewi-font";
+    rev = "21a86aa5fd14f7d8ee2a4eab454cca21d11b61c1";
+    sha256 = "daa1d12f279814ce74ef92b38f8093df64f41206b5b3f1b924b2d48dec56b542";
+  };
+
+  buildInputs = [ bdftopcf mkfontdir mkfontscale ];
+  buildPhase = ''
+    for i in *.bdf; do
+        bdftopcf -o $(echo -n $i | sed 's/bdf/pcf/') $i
+    done
+
+    gzip *.pcf
+  '';
+
+  installPhase = ''
+    fontDir="$out/share/fonts/misc"
+    mkdir -p "$fontDir"
+    mv *.pcf.gz "$fontDir"
+
+    cd "$fontDir"
+    mkfontdir
+    mkfontscale
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A nice bitmap font, readable even at small sizes";
+    longDescription = ''
+      Tewi is a bitmap font, readable even at very small font sizes. This is
+      particularrily usefull while programming, to fit a lot of code on your
+      screen.
+    '';
+    homepage = "https://github.com/lucy/tewi-font";
+    license = {
+      fullName = "GNU General Public License with a font exception";
+      url = "https://www.gnu.org/licenses/gpl-faq.html#FontException";
+    };
+    maintainers = [ maintainers.fro_ozen ];
+    platforms = platforms.linux;
```

Comment:
It should be safe to put `platforms.unix` here since basically all we need is stdenv and some utilities provided by Xorg.